pkg/handlers/registry.go

```diff
@@ -169,24 +169,9 @@ func (h *Handler) UpdateAppRegistry(w http.ResponseWriter, r *http.Request) {
 			skipImagePush = true
 		}
 
-		downstreams, err := store.GetStore().ListDownstreamsForApp(foundApp.ID)
+		appVersions, err := store.GetStore().FindAppVersions(foundApp.ID)
```

Comment:
we should stick to a consistent name for this variable. here it's called `downstreamVersions`: https://github.com/replicatedhq/kots/pull/2334/files#diff-a7b1ffb8e0d64772914b79d513e230636436763524feeee3e17d06925b2e15edR22

Reply:
and i think the name should be `appVersions`